iothub/device/src/ClientSettings/MqttTransportSettings.cs

```diff
@@ -17,7 +17,7 @@ public class MqttTransportSettings : ITransportSettings
 
         private const bool DefaultCleanSession = false;
         private const bool DefaultHasWill = false;
-        private const int DefaultKeepAliveInSeconds = 300;
+        private readonly TimeSpan DefaultKeepAlive = TimeSpan.FromSeconds(300);
```

Comment:
Convert to minutes for easier comprehension, given the scale of the value for seconds.
```suggestion
        private readonly TimeSpan DefaultKeepAlive = TimeSpan.FromMinutes(5);
```